src/cpp/addressanycommand.cpp

```diff
@@ -41,8 +41,10 @@ CommandReply AddressAnyCommand::run_me(RedisServer* server)
 Command* AddressAnyCommand::clone()
 {
     AddressAnyCommand* new_cmd = new AddressAnyCommand(*this);
-    if (new_cmd == NULL){
+    if (new_cmd == NULL) {
         throw std::bad_alloc();
     }
     return new_cmd;
-}
\ No newline at end of file
+}
+
+// EOF
```

Comment:
I might skip the ``// EOF `` comment since we aren't doing it in other files.

Reply:
I think I added it to all the files